brave/browser-laptop
# Rename about:style to about:styles

Follow up for https://github.com/brave/browser-laptop/pull/5242, per discussion on Slack

Auditors: @jkup

Test Plan:
- open new tab, type about:styles, verify it loads
- visit about:about + click about:styles, verify that loads
- run `npm run test -- --grep="about:styles"`
